deptofdefense/move.mil
# Add Feedback Form

## Checklist

I have…


## Summary of Changes

This pull request adds a user-facing Feedback Form (available at `/feedback`). The implementation uses Rails' [Action Mailer](http://guides.rubyonrails.org/action_mailer_basics.html) and a single view to generate a plaintext email including the user's feedback.

## Testing

To verify the changes proposed in this pull request…

1. clone this repo,
1. `git checkout add-feedback-form`,
1. set up development dependencies according to `CONTRIBUTING.md`,
1. run `brew services run mailhog` (if it's not already running),
1. run `bin/rails server`,
1. load up http://localhost:3000/feedback, fill out and submit the form, and
1. load up http://localhost:8025 and observe a received email!

## Screenshots

<details>
  <summary>Initial Screen</summary>

  ![narrow](https://user-images.githubusercontent.com/27780860/36733557-fad4e736-1b9e-11e8-9d2d-7d461e6ab163.png)

  ![wide](https://user-images.githubusercontent.com/27780860/36733562-fda67a10-1b9e-11e8-96c6-6b51c7f41020.png)
</details>

<details>
  <summary>Thank You Screen</summary>

  ![thanks-narrow](https://user-images.githubusercontent.com/27780860/36733568-00cde1ec-1b9f-11e8-89a8-7d780c5bc294.png)

  ![thanks-wide](https://user-images.githubusercontent.com/27780860/36733571-0393f33a-1b9f-11e8-82ad-ce60477b6ca0.png)
</details>
